deploy/test-environments/fleet_api/src/package_policy.py

```diff
@@ -0,0 +1,307 @@
+"""
+This module provides functions for generating and formatting policy templates
+and inputs based on provided data and templates.
+"""
+
+import copy
+import uuid
+from typing import Dict, Tuple
+from urllib.parse import urlparse, parse_qs
+from packaging import version
+from munch import Munch
+from loguru import logger
+from api.common_api import get_package
+
+# Constants
+CLOUD_SECURITY_POSTURE = "cloud_security_posture"
+REQUIRE_VARS = ["cloudbeat/cis_aws", "cloudbeat/cis_eks"]
+SIMPLIFIED_PACKAGE_POLICY = {
+    "policy_id": "",
+    "package": {},
+    "name": "",
+    "description": "",
+    "namespace": "default",
+    "inputs": {},
+    "vars": {},
+}
+SIMPLIFIED_AGENT_POLICY = {
+    "name": "",
+    "namespace": "default",
+    "monitoring_enabled": ["logs", "metrics"],
+}
+VERSION_MAP = {
+    "cis_k8s": "1.1.0",
+    "cis_eks": "1.2.0",
+    "cis_aws": "1.2.0",
+    "vuln_mgmt_aws": "1.3.0",
+    "cis_gcp": "1.5.0",
+}
+
+
+def generate_input_id(name: str, input_type: str) -> str:
+    """
+    Generates a unique input ID based on the provided name and input type.
+
+    This function combines the 'name' and 'input_type' parameters to create a unique
+    identifier for an input. The resulting ID is useful for organizing and referencing
+    inputs in a structured manner.
+
+    Args:
+        name (str): The name or identifier associated with the input.
+        input_type (str): The type or category of the input.
+
+    Returns:
+        str: A unique input ID generated by combining 'name' and 'input_type'.
+
+    Example:
+        If 'name' is "cspm" and 'input_type' is "cloudbeat/cis_gcp", calling
+        'generate_input_id(name, input_type)' will return "cspm-cloudbeat/cis_gcp".
+    """
+    return f"{name}-{input_type}"
```

Comment:
This entire function is just concating two strings, nothing is unique about the resulting ID. I think we can remove it. Seeing the contamination itself in a code block would be more readable then calling this function IMO.